pkg/kind/kind.go

```diff
@@ -316,5 +343,5 @@ func parseKindVersion(v string) (float64, error) {
 }
 
 func deleteContainerRegistry() *exec.Cmd {
-	return exec.Command("docker", "rm", "-f", container_reg_name, "&&", "||", "true")
+	return exec.Command(container_engine, "rm", "-f", container_reg_name, "&&", "||", "true")
```

Comment:
```suggestion
	return exec.Command(container_engine, "rm", "-f", container_reg_name)
```

doesn't look like the delete function actually runs otherwise...

Reply:
I think it does not run because it try to combine both `&&` and `||`, the result command is something like: `podman rm -f <...> && || true` when it should be `podman rm -f <...> && true` or `podman rm -f <...> || true`. But it works without any of these options because we are handling the errors when calling the `deleteContainerRegistry()` function.